lib/flags.ml

```diff
@@ -6,18 +6,28 @@
 (*         *       GNU Lesser General Public License Version 2.1        *)
 (************************************************************************)
 
-let with_modified_ref r nf f x =
+(* [respect] indicates whether if [f] modifies the ref, we preserve
```

Comment:
I think you mean `restore`, not `respect` in the comment

Reply:
Yup, the option was renamed in some revision of the patch, both comments fixed, thanks!